AirCasting/APICommunicator/APIcalls.swift

```diff
@@ -0,0 +1,33 @@
+// Created by Lunar on 01/07/2021.
+//
+
+import Foundation
+
+final class APIcalls {
```

Comment:
Could a change a name to sth more descriptive? 
Or maybe it could be an extension to AuthorizationApi, what do you think @pawelgil  @mateuszmackowiak ?

Reply:
Maybe we can move this func to AuthorizationAPI? Not sure, I think that it's logical to have it there but maybe @pawelgil  share his opinion? 
If not I'd still call it differently than "Apicalls" or "APICommunication". In those two we could put all the API communication functions. Can you rename it to just ForgotPasswordAPI? It would be consistent with AuthorizationAPI and CreateSessionAPI :)